### DISCUSSION TITLE
About startup

### DISCUSSION THREAD

**Discussion by raghu-007** [EddieHubCommunity/support#6962] [General] [answered]
Is it a nice idea to open source a startup??

**Comment by ayush24k** [comment, marked as answer]
According to me the answer is both "YES" and "NO". If your startup is unique and using a unique functionality that is not easy to replicate it is better keep the code private and If the idea is make a change make it open source so everyone can contribute and make work easier

**Comment by raghu-007** [comment]
Yup it's true tq 😃
